aapcs64/aapcs64.rst

```diff
@@ -3054,35 +3066,37 @@ document.
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVInt16_t``                | ``svint16_t``         | signed half-word                          | VG×4           |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVUint16_t``               | ``svuint16_t``        | unsigned half-word                        | VG×4           |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVFloat16_t``              | ``svfloat16_t``       | half-precision float                      | VG×4           |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVBfloat16_t``             | ``svbfloat16_t``      | half-precision brain float                | VG×4           |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVInt32_t``                | ``svint32_t``         | signed word                               | VG×2           |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVUint32_t``               | ``svuint32_t``        | unsigned word                             | VG×2           |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVFloat32_t``              | ``svfloat32_t``       | single-precision float                    | VG×2           |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVInt64_t``                | ``svint64_t``         | signed double-word                        | VG             |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVUint64_t``               | ``svuint64_t``        | unsigned double-word                      | VG             |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVFloat64_t``              | ``svfloat64_t``       | double-precision float                    | VG             |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | ``__SVBool_t``                 | ``svbool_t``          | single bit (fully packed into VG bytes)   | VG×8           |
+  +--------------------------------+-----------------------+-------------------------------------------+----------------+
+  | **(Alpha)** ``__SVMfloat8_t``  | ``svmfloat8_t``       | modal 8-bit float                         | VG×8           |
```

Comment:
Not a problem of this patch, but at some point we should add "svcount" if ABI is used to described the types